### ISSUE TITLE
Only AdMob Rewarded ads getting ad fill in Mediated Test Suite (Unity 2018.2.7f1, iOS)

### ISSUE THREAD

**Issue by ashbash1987** [googleads/googleads-mobile-unity#1055]
Have the following version configuration of AdMob and some mediated ad providers (and test suite):
- Google Mobile Ads UnityPackage 4.1.0 (Mobile Ads SDK 7.51.0)
- AdColony UnityPackage 1.0.5 (iOS adapter 3.3.7.2 from Pods)
- Chartboost UnityPackage 2.0.0 (iOS adapter 8.0.1.1 from Pods)
- Facebook UnityPackage 2.5.1 (iOS adapter 5.5.1.0 from Pods)
- Unity UnityPackage 2.2.0 (iOS adapter 3.2.0.1 from Pods)
- Vungle UnityPackage 3.1.4 (iOS adapter 6.3.2.3 from Pods)
- TestSuite UnityPackage 1.1.0

Using the mediated ads test suite with a valid test device ID, I can successfully verify that advertising works for an Interstitial ad placement for all of the networks listed. However, for a Rewarded ad placement, only AdMob is successfully getting ad fill.

Further, when I attempt to load a rewarded ad from anything other than AdMob, I get this in the log console from the device:
```
<Google> Ad network adapter GADMAdapterUnity implements an old API. Please update the adapter.
<Google> Ad network adapter GADMAdapterChartboost implements an old API. Please update the adapter.
[... etc for the other network adapters ..]
```
I don't get a log message like the above for interstitials though - just rewarded ads, which is a bit strange.

I've also double checked all the mediation setup with all the providers and the AdMob site, and everything seems to be pointing to the correct ad units and locations.

As far as I can see, all of the versions above of the Unity packages from the main doc pages on Unity ad mediation with Google Mobile Advertising are the latest versions available from the BinTray; as such, I would expect the Pod dependencies they declare should be valid and "up-to-date" as it were, and I shouldn't need to adapt the dependency files after the face to get more recent versions of the adapters (if such updated versions exist).

Am I going mad, or is there something blindingly obvious that I'm missing? Or is it that the mediated test suite is maybe trying to make an old-style rewarded API request? (completely spitballing ideas) 😕 

_(Sidenote, we've verified that the exact same Unity packages mentioned work perfectly for Android across all ad units and ad networks)_

**Comment by ashbash1987** [comment]
An update to this.

I do believe that this is an issue with the _test suite_ only, as I've now been able to verify getting adverts from the mediated ad networks from the rewarded ad unit in the game itself, and still not  through the test suite. Also of note, I do not get a log message indicating "old API" usage while requesting the rewarded ads in the standard way in the game, while they still persist in the test suite (for rewarded ads).

**Comment by stowy** [comment]
Thanks for reporting, yes I can confirm that there is a bug in loading rewarded ads in the current release of mediation test suite for iOS. This is not indicative of a problem in your app. There will be an update published in the next week or so to address this. In the meantime if the other checks are passing and you are also seeing ad requests for the network in question firing using charles or your console logs for example, then it's reasonable to assume that your implementation is correct.

**Comment by DongerGroup** [comment]
I got the same problem,To AdColony,got this problem in the latest verison of admobtestsuite plugin
![image](https://user-images.githubusercontent.com/35094427/68603408-2995ec80-04e3-11ea-91b7-06d408766712.png)

**Comment by stowy** [comment]
I've released the fix in iOS in version 1.1.1 of the Mediation Test Suite. You can update the dependencies file in the plugin to receive the fix. Alternatively I will be releasing an update to the MTS unity package in the coming days.

**Comment by stowy** [comment]
Released version 1.2.1 of the Mediation Test Suite Unity plugin which contains the iOS update.
